doc/FAQ.md

```diff
@@ -131,9 +131,109 @@ Tools for replication are currently in active development, using the `db_write`
 [plugin hook](https://lightning.readthedocs.io/PLUGINS.html#db-write).
 
 
-## Loss
+## Channel Management
+
+### How to forget about a channel? <a name="forget-channel"></a>
+
+Channels may end up stuck during funding and never confirm
+on-chain. There is a variety of causes, the most common ones being
+that the funds have been double-spent, or the funding fee was too low
+to be confirmed. This is unlikely to happen in normal operation, as
+CLN tries to use sane defaults and prevents double-spends whenever
+possible, but using custom feerates or when the bitcoin backend has no
+good fee estimates it is still possible.
+
+Before forgetting about a channel it is important to ensure that the
+funding transaction will never be confirmable by double-spending the
+funds. To do so you have to rescan the UTXOs using
+[`dev-rescan-outputs`](#rescanning) to reset any funds that may have
+been used in the funding transaction, then move all the funds to a new
+address:
+
+```bash
+lightning-cli dev-rescan-outputs
+ADDR=$(lightning-cli newaddr bech32 | jq .bech32)
+lightning-cli withdraw $ADDR all
+```
+
+This step is not required if the funding transaction was already
+double-spent, however it is safe to do it anyway, just in case.
+
+Then wait for the transaction moving the funds to confirm. This
+ensures any pending funding transaction can no longer be
+confirmed. 
+
+As an additional step you can also force-close the unconfirmed channel:
+
+```bash
+lightning-cli close $PEERID 10  # Force close after 10 seconds
+```
+
+This will store a unilateral close TX in the DB as last resort means
+of recovery should the channel unexpectedly confirm anyway.
+
+Now you can use the `dev-forget-channel` command to remove
+the DB entries from the database.
+
+```bash
+lightning-cli dev-forget-channel $NODEID
```

Comment:
Needs a note that to access the `dev-` commands you need to be using a developer build.

Reply:
Added a quick notice about the `dev-` commands only being available in `DEVELOPER=1` builds